test/features/support_archive/files.go

```diff
@@ -5,6 +5,7 @@ package support_archive
 import (
```

Comment:
When trying to run the e2e tests
```sh
❯ make test/e2e/supportarchive --debug
GNU Make 4.3
Built for x86_64-pc-linux-gnu
Copyright (C) 1988-2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Reading makefiles...
Updating makefiles....
Updating goal targets....
 File 'test/e2e/supportarchive' does not exist.
   File 'manifests/crd/helm' does not exist.
     File 'prerequisites/kustomize' does not exist.
    Must remake target 'prerequisites/kustomize'.
go install "sigs.k8s.io/kustomize/kustomize/v5@v5.3.0"
    Successfully remade target file 'prerequisites/kustomize'.
     File 'helm/version' does not exist.
    Must remake target 'helm/version'.
    Successfully remade target file 'helm/version'.
     File 'manifests/crd/generate' does not exist.
       File 'prerequisites/controller-gen' does not exist.
      Must remake target 'prerequisites/controller-gen'.
go install "sigs.k8s.io/controller-tools/cmd/controller-gen@v0.13.0"
      Successfully remade target file 'prerequisites/controller-gen'.
    Must remake target 'manifests/crd/generate'.
/home/alberto/go/bin/controller-gen "crd:crdVersions=v1" paths="./..." output:crd:artifacts:config=config/crd/bases
    Successfully remade target file 'manifests/crd/generate'.
  Must remake target 'manifests/crd/helm'.
./hack/helm/generate-crd.sh /home/alberto/go/bin/kustomize config/helm/chart/default//templates//Common/crd/ config/deploy/
# Warning: 'patchesStrategicMerge' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.
  Successfully remade target file 'manifests/crd/helm'.
Must remake target 'test/e2e/supportarchive'.
go test -v -tags "osusergo,netgo,sqlite_omit_load_extension,e2e," -timeout 20m -count=1  ./test/scenarios/standard -args --labels "name=support-archive"
F1214 14:46:34.322841  168239 env.go:375] Setup failure: exit status 2
FAIL	github.com/Dynatrace/dynatrace-operator/test/scenarios/standard	3.784s
FAIL
make: *** [hack/make/tests/e2e.mk:101: test/e2e/supportarchive] Error 1
```

Reply:
remove `dynatrace` namespace before running the test

Reply:
Passing now! Thanks